Description: Restore libsuffix support in pkg-config file
 This is needed for 64-bit flavours.
Author: Sébastien Villemot <sebastien@debian.org>
Bug-Debian: https://bugs.debian.org/1087874
Forwarded: https://github.com/OpenMathLib/OpenBLAS/pull/4982
Last-Update: 2024-11-20
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/openblas.pc.in
+++ b/openblas.pc.in
@@ -2,6 +2,6 @@ Name: openblas
 Description: OpenBLAS is an optimized BLAS library based on GotoBLAS2 1.13 BSD version
 Version: ${version}
 URL: https://github.com/xianyi/OpenBLAS
-Libs: -L${libdir} -l${libprefix}openblas${libnamesuffix}
+Libs: -L${libdir} -l${libprefix}openblas${libnamesuffix}${libsuffix}
 Libs.private: ${extralib}
 Cflags: -I${includedir}
